In [1]:
#import the libraries
import pandas as pd
import scipy 
import numpy as np
from scipy import stats

In [2]:
#import data
data = pd.read_csv('BuyerRatio.csv')

In [4]:
#display data
data

Observed Values  East  West  North  South
0           Males    50   142    131     70
1         Females   435  1523   1356    750

In [6]:
#load data from table to array required for chi2 analysis
data_obs=np.array([[50,142,131,70],[435,1523,1356,750]])
data_obs

array([[  50,  142,  131,   70],
       [ 435, 1523, 1356,  750]])

In [8]:
# Chi2 contengency independence test with output as (Chi2 stats value, p_value, df, expected obsvations)
p_value = stats.chi2_contingency(data_obs)

In [10]:
p_value[1]

0.6603094907091882

The calculated P_value is 0.660 > 0.05 hence accept null hypothesis and the proportion is similar for male and female accross region but not related